hoodie-client/src/main/java/com/uber/hoodie/HoodieWriteClient.java

```diff
@@ -452,6 +452,7 @@ public boolean savepoint(String commitTime, String user, String comment) {
             .getHoodieTable(new HoodieTableMetaClient(fs, config.getBasePath(), true), config);
         Optional<HoodieInstant> cleanInstant = table.getCompletedCleanTimeline().lastInstant();
 
+        // NOTE(na): MOR has deltacommit action
         HoodieInstant commitInstant = new HoodieInstant(false, HoodieTimeline.COMMIT_ACTION, commitTime);
```

Comment:
This always assumes a savepoint on a COMMIT_ACTION, but as a part of MOR, we will have commits and/or deltacommits.